### PR TITLE
add effect tests

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: libs/category/state/src/effects/category-page-filter.effects.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class DaffCategoryPageFilterEffects<
    * in a sequence.
    */
   @Effect()
-  updateFilters$: (debounceFrame: number) => Observable<any> = (debounceFrame = 300) => this.actions$.pipe(
```
Comment (damienwebdev) [inline libs/category/state/src/effects/category-page-filter.effects.ts:71]: No explicit any.

File: libs/category/state/src/effects/category-page-filter.effects.spec.ts [hunk 2]
```diff
@@ -26,158 +31,387 @@ import {
...
+        });
       });
     });
   });
 
-  describe('when ToggleCategoryFilterAction is triggered', () => {
+  describe('when CategoryPageApplyFiltersAction is triggered', () => {
+    let expected;
+
+    beforeEach(() => {
```
Comment (damienwebdev) [inline libs/category/state/src/effects/category-page-filter.effects.spec.ts:266]: These are all the same test, can they be merged into a single function and then called for each action type?

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/category/state/src/effects/category-page-filter.effects.ts (hunk 1 above)
Comment (griest024) [reply libs/category/state/src/effects/category-page-filter.effects.ts:71]: typed as the explicit actions

File: libs/category/state/src/effects/category-page-filter.effects.spec.ts (hunk 2 above)
Comment (griest024) [reply libs/category/state/src/effects/category-page-filter.effects.spec.ts:242]: made it into 2 functions and called each

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**